pylint_pytest/checkers/fixture.py

```diff
@@ -25,6 +25,19 @@ class FixtureChecker(BasePytestChecker):
             'deprecated-pytest-yield-fixture',
             'Used when using a deprecated pytest decorator that has been deprecated in pytest-3.0'
         ),
+        'W6402': (
+            'Using useless `@pytest.mark.usefixtures` decorator for fixtures',
+            'useless-pytest-usefixtures-decorator',
+            (
+                '@pytest.mark.usefixtures decorator doesn\'t work as expected for fixtures. '
+                'Take a look at: https://docs.pytest.org/en/stable/fixture.html#usefixtures'
+            ),
+        ),
+        'W6403': (
+            'Using a deprecated scope of fixture as positional argument',
+            'deprecated-pytest-fixture-scope-as-positional-argument',
+            'Pass scope as a kwarg, not positional arg, which is deprecated in future pytest.',
```

Comment:
All positional arguments, not just `scope`, are prohibited since pytest 6.0.0 https://docs.pytest.org/en/stable/deprecations.html#pytest-fixture-arguments-are-keyword-only